### ISSUE TITLE
Include .npmrc file

### ISSUE THREAD

**Issue by rept** [getkuby/kuby-core#53]
I'm using Font Awesome pro and have set the key it uses in the .npmrc file (in the root of the project).  (see here for more info: https://fontawesome.com/how-to-use/on-the-web/setup/using-package-managers)

Locally the yarn install is working fine.  However when building the docker it's giving an unauthenticated error.  Probably because that file isn't copied into the container.  Is there a way to force it to copy that before executing the yarn command?

**Comment by rept** [comment]
It works if I add this line.  Not sure if that's best practise though.

```
module Kuby
  module Docker
    class YarnPhase < Layer
      extend T::Sig

      sig { params(dockerfile: Dockerfile).void }
      def apply_to(dockerfile)
        dockerfile.copy('package.json', '.')
        dockerfile.copy('yarn.lock*', '.')
        dockerfile.copy('.npmrc', '.')
        dockerfile.run('yarn', 'install')
      end
    end
  end
end
```

**Comment by camertron** [comment]
@rept hmm that's weird. If I add .npmrc to a project and run `kuby build`, I can see it got copied into the container. Do you see it if you run `docker run --rm -it <image url>:latest /bin/ls -lah`?

**Comment by rept** [comment]
I've changed the code above and it compiled fine after that, before it didn't (I tried several times).  Does it get copied over before the yarn install command?  Otherwise it's too late.

**Comment by camertron** [comment]
Hey @rept, I totally misread your earlier posts. You're absolutely right, .npmrc (and .yarnrc for that matter) are only copied over in the copy phase, which happens after the yarn phase, meaning they won't be available when `yarn install` runs. I have made the changes to kuby-core and published v0.11.15 of the gem. Thanks!

**Comment by camertron** [comment]
Oops, make that v0.11.16. Did the COPY wrong.
